binding/react-native/ios/Porcupine.swift

```diff
@@ -83,7 +96,50 @@ class PvPorcupine: NSObject {
             resolve(keywordIndex)
         }
         else{
-            reject("PvPorcupine:process", "Invalid Porcupine handle provided to native module.", nil)
+            reject(pvStatusToExceptionCode(PV_STATUS_INVALID_STATE), "Invalid Porcupine handle provided to native module.", nil)
+        }
+    }
+
+    private func getResourcePath(_ filePath: String) throws -> String {
+        if (!FileManager.default.fileExists(atPath: filePath)) {
+            if let resourcePath = Bundle.main.resourceURL?.appendingPathComponent("resources/\(filePath)").path {
+                if (FileManager.default.fileExists(atPath: resourcePath)) {
+                    return resourcePath
+                }
+            }
+            
+            throw PvPorcupineError.RuntimeError("Could not find file at path '\(filePath)'. If this is a packaged asset, ensure you have added it to your xcode project.")
```

Comment:
It seems unnecessary to have that custom error for only this line. Maybe just throw a RuntimeError here and remove the custom error altogether

Reply:
I am not really sure there are 'default' error types in swift? I tried them as well search for it. don't think they exist.